src/utilities/MonteCarlo/_UnitTests/test_scenarioBasicOrbitMC.py

```diff
@@ -70,10 +71,18 @@ def myCreationFunction():
     # Setup the simulation modules
     # Initialize spacecraft object and set properties
     scObject = spacecraft.Spacecraft()
-    scObject.ModelTag = "bsk-Sat"
+    scObject.ModelTag = "bskSat"
     # Add spacecraft object to the simulation process
     sim.AddModelToTask(simTaskName, scObject)
 
+    # Set up SimpleTestModule to test a getter function using AddVariableForMultiProcessLogging
+    testModule = SimpleTestModule()
+    testModule.ModelTag = "helloworldModule"
+    sim.AddModelToTask(simTaskName, testModule)
+
+    sim.AddVariableForMultiProcessLogging("helloworldModule.GetTicker()")
+    sim.AddVariableForMultiProcessLogging("bskSat.totOrbEnergy")
```

Comment:
:large_orange_diamond: Should this test assert some

Reply:
Yes, currently they are just being checked to see if they run at all but I will look into seeing what we can assert. Likely making sure the shape of the output variable is greater than 2x0

Reply:
This was a great call because I totally realized I shouldnt have been adding logs to the sim directly, we now test in the same way as Messages and looks much better